symfony/src/Controller/WorkQueueController.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+namespace App\Controller;
+
+use App\Service\LoggerService;
+use App\Service\SiteService;
+use App\Service\WorkQueueService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+use Symfony\Component\Routing\Annotation\Route;
+use App\Helper\WorkQueue;
+use Pmi\Audit\Log;
+
+/**
+ * @Route("/s/workqueue")
+ */
+class WorkQueueController extends AbstractController
+{
+    /**
+     * @var SessionInterface
+     */
+    protected $session;
+    /**
+     * @var WorkQueueService
+     */
+    protected $workQueueService;
+    /**
+     * @var SiteService
+     */
+    protected $siteService;
+
+    /**
+     * WorkQueueController constructor.
+     * @param SessionInterface $session
+     * @param WorkQueueService $workQueueService
+     * @param SiteService $siteService
+     */
```

Comment:
I noticed in this PR you started introducing more docblocks with `@param` and `@return` lines. We can discuss more, but in my opinion, we should use native parameter types and return types instead.